peachjam/js/components/FindDocuments/index.vue

```diff
@@ -207,6 +207,7 @@
       {{ $t('Court registry') }}
       {{ $t('Judges') }}
       {{ $t('Attorneys') }}
+      {{ $t('Order outcome') }}
```

Comment:
In the detail view, we use `Order`. Let's be consistent with the `Order` label, rather than `Order outcome`. The internal field name remains as `order_outcome`.